pom.xml

```diff
@@ -124,5 +124,33 @@
         </plugins>
     </reporting>
 
+    <profiles>
+        <profile>
+            <id>openrewrite</id>
```

Comment:
Technically we don't absolutely need a profile for the workflows to work; but by having a profile here we more easily can bump the versions of the plugin and dependency, and folks can run the same fixes locally with just:
```bash
mvn --activate-profiles openrewrite org.openrewrite.maven:rewrite-maven-plugin:run
```

Reply:
I think having a dedicated profile is the right way.
As maven documentation states "Under other circumstances, a slightly different dependency set will be required, (...)" and that's the situation here.

Maybe chaining the plugin execution to a specific build step (verify) would be even more maven-ish.

Reply:
Thanks! You mean to only add it to a build step in this particular profile? That could make sense indeed. I hadn't wanted to tie it into the lifecycle originally, as to not cause delays for folks, but within a profile that makes sense.